makefile

```diff
@@ -11,7 +11,7 @@ ifeq ($(UNAME), Darwin) # macOS
 	LDFLAGS += -L/usr/local/include -lboost_system -lboost_filesystem
 endif
 
-DEBUG_FLAGS = -g -Wall -Werror
+DEBUG_FLAGS = -g -Wall -Wextra -Werror
```

Comment:
Okay, I looked this up and I guess this is good to have... Don't know if this would cause longer coding time though in terms of efficiency. Your thoughts? @evangrayk